lib/assessment.js

```diff
@@ -40,8 +58,12 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 // the object passed in as an argument or the object created
 var addMethod = function addMethod(propertyName, method, obj) {
 
+//i have no idea how to do this on my own
 };
 
+//just FYI, I noticed yesterday that the solution branch was accessible to us (didnt look)
+//It might be beneficial to keep it private until after we take the assessment so we aren't tempted to look at the answers prior to the assessment
+
```

Comment:
We're going by the honor system. People are welcome to look at the answers... it just won't help them (or us) accurately assess their understanding. And at the end of the day, that's the only real point of the assessments anyway.